<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BudgetMgrEditClass</class>
 <widget class="QDialog" name="BudgetMgrEditClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>157</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>559</width>
    <height>337</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BudgetMgrEdit</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="Label_3">
      <property name="text">
       <string>预算日期:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDateEdit" name="budgetDate">
      <property name="minimumDate">
       <date>
        <year>1752</year>
        <month>9</month>
        <day>14</day>
       </date>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="date">
       <date>
        <year>2000</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>预算编号:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="budgetId"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_4">
    <item row="0" column="0">
     <widget class="QLabel" name="Label">
      <property name="text">
       <string>科目:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="budgetTitle">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="Label_2">
      <property name="text">
       <string>详细科目</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="budgetSubtitle">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_5">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>117</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnSave">
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="Label_4">
      <property name="text">
       <string>预算金额:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="budgetAmount"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>元</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>94</y>
     <width>461</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>备注:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="budgetMemo"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnSave</sender>
   <signal>clicked()</signal>
   <receiver>BudgetMgrEditClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>BudgetMgrEditClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
